ui/src/App.tsx

```diff
@@ -321,82 +304,116 @@ const AppComponent = () => {
-        render: () => (
-          <ServiceRequired service={ServiceKind.AUCTION} action="New Auction">
-            <NewAuction services={auctionService} />
-          </ServiceRequired>
-        ),
-      },
-      {
-        path: paths.app.setup.identity,
-        render: () => <RequestIdentityVerification />,
+        path: paths.app.instruments.instrument + '/:contractId',
+        component: Instrument,
       },
       {
-        path: paths.app.setup.instrument.new.base,
+        path: paths.app.instruments.new.base,
         render: () => (
           <ServiceRequired service={ServiceKind.ISSUANCE} action="New Base Instrument">
             <NewBaseInstrument />
           </ServiceRequired>
         ),
       },
       {
-        path: paths.app.setup.instrument.new.convertiblenote,
+        path: paths.app.instruments.new.convertiblenote,
         render: () => (
           <ServiceRequired service={ServiceKind.ISSUANCE} action="New Convertible Note">
             <NewConvertibleNote />
           </ServiceRequired>
         ),
       },
       {
-        path: paths.app.setup.instrument.new.binaryoption,
+        path: paths.app.instruments.new.binaryoption,
         render: () => (
           <ServiceRequired service={ServiceKind.ISSUANCE} action="New Binary Option">
             <NewBinaryOption />
           </ServiceRequired>
         ),
       },
+    ],
+  });
+
+  const issuanceCustomer = issuanceService.filter(cs => cs.payload.customer === party);
+
+  entries.push({
+    displayEntry: () => issuanceCustomer.length > 0,
+    sidebar: [
       {
-        path: paths.app.setup.issuance.new,
+        label: 'Issuance',
+        activeSubroutes: true,
+        path: paths.app.issuance.root,
+        render: () => <IssuancesTable />,
+        icon: <ControlsIcon />,
+        children: [],
+      },
+    ],
+    additionalRoutes: [
+      {
+        path: paths.app.issuance.new,
         render: () => (
           <ServiceRequired service={ServiceKind.ISSUANCE} action="New Issuance">
             <IssuanceNew services={issuanceService} />
           </ServiceRequired>
         ),
       },
+    ],
+  });
+  entries.push({
+    displayEntry: () => true,
+    sidebar: [
+      {
+        label: 'Trading',
+        activeSubroutes: true,
+        path: paths.app.trading,
+        render: () => <TradingServiceTable services={tradingService} />,
+        icon: <ControlsIcon />,
+        children: [],
+      },
+    ],
+  });
+
+  // additionalRoutes: [
+  //   {
+  //     path: paths.app.listings + '/:contractId?',
+  //     render: () => <ListingsTable services={listingService} listings={listings} />,
+  //   },
+  //   {
+  //     path: paths.app.listings.new,
+  //     render: () => (
+  //       <ServiceRequired service={ServiceKind.LISTING} action="New Listing">
+  //         <ListingNew services={listingService} />
+  //       </ServiceRequired>
+  //     ),
+  //   },
+  // ],
```

Comment:
So I could only find Listing routes from other listing routes. Should it be in the sidebar? Or under trading? @alexgraham-da

Reply:
I think Listings would work in the sidebar. It's technically different than Trading in terms of the services